src/beast/beast/nudb/win32_file.h

```diff
@@ -42,6 +42,9 @@
 # ifndef STRICT
 #  define STRICT
 # endif
+# ifndef WIN32_LEAN_AND_MEAN
```

Comment:
This is already defined in _ten_ different places in the code - do we really need an eleventh?